engine/gamesys/src/gamesys/components/comp_sprite.cpp

```diff
@@ -65,6 +66,31 @@ namespace dmGameSystem
         SpriteResourceOverrides() : m_Material(0) {}
     };
 
+    const uint32_t MAX_TEXTURE_COUNT = dmRender::RenderObject::MAX_TEXTURE_COUNT;
+    const uint8_t CACHE_EVICTION_FRAMES = 10; // how many frames cache entry is stored before eviction
+
+    struct AnimationData
```

Comment:
I'd also like a code comment about how the animation data usage is intended to work and why.
This component is getting quite complex, and I think we'll need more documentaion moving on.